bot/exts/easter/conversationstarters.py

```diff
@@ -3,13 +3,23 @@
 import random
 from pathlib import Path
 
+from discord import Embed
 from discord.ext import commands
 
+from bot.utils.decorators import override_in_channel
+
+
 log = logging.getLogger(__name__)
 
+
 with open(Path("bot/resources/easter/starter.json"), "r", encoding="utf8") as f:
     starters = json.load(f)
```

Comment:
This is a constant, so the naming should follow `UPPER_SNAKE_CASE`.
```suggestion
    STARTERS = json.load(f)
```